### PR TITLE
A bunch of traitor items

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

- Gave the motherbase shipment a dumping in price + added padding to the kit, gave the cqc arm a proper sprite.
- Ported hippie butterfly knives (makeshift one and the energy variant for traitors)
- Added the Serrated Blade, a fairly expensive knife that has a 10% chance to score a critical hit and a 1% chance to cause neck slicing on the target (on blooded species, non-blooded ones simply receive 125 brute damage). This comes at a cost, since the damage is only 10 brute and it weighs more than a combat knife.
- Added Blackened Dagger. A combat knife, but weighs more and heals you for half the value of the damage caused on an attack on a mob.
- Added Mors Plant (traitor botanist exclusive). A knife, capable of destroying plant related atoms instantly, and causes neck slicing on plantpeople and mushroom people.
- Balanced molag bal mace and renamed it to Will Breaker. Now it costs a bit more, is more bulky, but has a higher chance of causing internal organ damage.
- Added a jammer pen. Essentially a more expensive radio jammer that fits in your pocket and acts like a pen.
- Added the mask-querade. Turns the user into a completely different person while worn. No mob overlay, can only be discovered by examination or unfitting PDA/ID.
- Added screamer bundle. Has a special knife with two modes of attack, a mask that essentially works like the mask-querade but has a proper overlay and always changes the name of the wearer to "Screamer", and armored black robes.

## Why It's Good For The Game

fun tator item

## Changelog
:cl: Bob Joga and Junglerat
add: Butterfly knives, traitor variant and makeshift variant.
add: Serrated blade traitor item.
add: Blackened dagger traitor item.
add: Mors Plant botanist traitor item.
add: Jammer pen traitor item.
add: Mask-querade traitor item.
add: Screamer traitor bundle.
balance: Upped the cost of the mace of molag bal, made it bulky, but made it deal more "special" damage. Changed the name to "Will Breaker".
balance: Motherbase equipment bundle cost lowered. Added melee padding to the bundle.
imageadd: Proper icons for the CQC arm.
/:cl:

### CODE REVIEW

**Comment by SarmentiCampbell** [comment]
Yes but that nettlebane seem way too OP, ; -;

**Comment by BobJoga** [comment]
> Yes but that nettlebane seem way too OP, ; -;

Really circumstantial. On most situations it's useless.

**Comment by ShadeAware** [comment]
Nettleblade shouldn't be a 1 hit KO, but otherwise I **really** enjoy these and the ideas behind them! 👍

**Comment by BobJoga** [comment]
> Nettleblade shouldn't be a 1 hit KO, but otherwise I **really** enjoy these and the ideas behind them! 👍

Changing it to just do the neck slice effect on plantpeople.

**Comment by Akrilla** [comment]
It's not in the scope of this PR so no worries, but I really like the idea of the mask-querade being more generic, and way more accessible to traitors. People metaing species features hurts a lot of stealth gameplay, and being able to easily change your species look "should" be standard for traitors in a codebase with so many custom species.

You could do this by reducing the cost of that item massively, or however else you think would work.

**Comment by BobJoga** [comment]
> It's not in the scope of this PR so no worries, but I really like the idea of the mask-querade being more generic, and way more accessible to traitors. People metaing species features hurts a lot of stealth gameplay, and being able to easily change your species look _should_ be standard for traitors in a codebase with so many custom species.
> 
> You could do this by reducing the cost of that item massively, or however else you think would work.

Mulligan might be a bit more violent and basically have irreversible effects, but i feel it does the job just fine. I will however lower the cost of the mask, since it really makes no sense for the screamer bundle to cost only 2tc more despite coming with two extra items that are actually very good.

**Comment by Akrilla** [comment]
Mull is too permanent for what I'm wanting I think, but for now TC decrease is good yeah.

File: modular_skyrat/code/game/objects/items/devices/traitordevices.dm [hunk 1]
```diff
@@ -1,7 +1,9 @@
+//morph cube
 GLOBAL_LIST_INIT(blacklistedmorphcubemobs, typecacheof(list(\
 										/mob/living/simple_animal/hostile/boss,\
-										/mob/living/carbon, /mob/living/simple_animal/hostile/asteroid/elite,\
-										/mob/living/simple_animal/hostile/boss, /mob/living/silicon,\
+										/mob/living/simple_animal/hostile/megafauna,\
```
Comment (Akrilla) [inline modular_skyrat/code/game/objects/items/devices/traitordevices.dm:4]: omfg thank you

**Comment by Akrilla** [comment]
Fixup conflicts and this is fine for merge.

**Pull request merged**